### ISSUE TITLE
Image Fetching Not Working

### ISSUE THREAD

**Issue by ravaliya** [DWilliames/google-sheets-sync-figma#45]
**Describe the bug**

I have a sheet with a couple of columns of text, and then links to images that are in my google drive. All images are set to fully open permissions. When i first tried to fetch and sync it worked. However, after that, anytime i try and fetch and sync i get an "Error fetching image". The text fields sync properly. but the image does not load and i get a pink checkerboard with red text that says "Error fetching image"

**To Reproduce**
Steps to reproduce the behavior: (include a screen recording if possible) and link to Figma file and Google Sheet file

This is the link to the sheet: https://docs.google.com/spreadsheets/d/17AKLRgIRLbK9Rm3wWQA-jOqCTWtDbMJkUKB1pKxR9oM/edit?usp=sharing

**Expected behavior**

I would expect the image to fetch and load in figma everytime.

**Additional context**

![image](https://user-images.githubusercontent.com/2974855/122972170-c1c6a980-d387-11eb-95d5-56d5142aefe8.png)

**Comment by hkutcher** [comment]
I have the same problem. Is Figma's auto layout the problem?

**Comment by ravaliya** [comment]
I don't think so, I've tested with and without auto layout. Also tested if they are component instances or detached as normal groups. still fails.

**Comment by newoldschool** [comment]
Hi, have you received any response from the developer? I'm having the same problem

**Comment by ten-tjc** [comment]
This seems to happen to me randomly. Then randomly it starts working again.

**Comment by djpecot** [comment]
Yep, stopped working then randomly starts working again for me as well. I tried checking console but when I re-synced with console open it mysteriously worked 😅
